python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/router-traffic-engineering.j2

```diff
@@ -42,6 +42,34 @@
 {%                 endfor %}
 {%             endfor %}
 {%         endif %}
+{%         if router_traffic_engineering.flex_algos is arista.avd.defined %}
+
+##### Flex-algo
+
+| Algo Number | Algo Name | Priority | Metric | Color | Admin-groups | SRLG Excludes |
+| ----------- | --------- | -------- | ------ | ----- | ------------ | ------------- |
+{%             for algo in router_traffic_engineering.flex_algos %}
+{%                 set priority = algo.priority | arista.avd.default("-") %}
+{%                 set metric = algo.metric | arista.avd.default("-") %}
+{%                 set color = algo.color | arista.avd.default("-") %}
+{%                 if algo.administrative_group is arista.avd.defined %}
+{%                     set admingrp = "" %}
+{%                     if algo.administrative_group.include_all is arista.avd.defined %}
+{%                         set admingrp = admingrp ~ "include-all " ~ algo.administrative_group.include_all | join(",") ~ " " %}
```

Comment:
This is fragile when the user combines ranges and individual items. Maybe we just take a single string instead like we do for `switchport vlan` etc.

Reply:
True, i have changed it to plain str instead.